CONTRIBUTING.md

```diff
@@ -24,6 +24,7 @@ Clone & install dependencies:
 git clone https://github.com/amelioro/ameliorate.git
 cd ameliorate
 npm install # repo-wide dependencies
+npx husky install # Activate commit hooks
```

Comment:
Can we actually move this to the Conventional Commits section? Since it's not necessary for running the project (similarly, the `npm run lint` command is mentioned in the Code Style section)

Reply:
sure can! 
Tried to amend this commit and I'm getting this error:
`sh: 1: commitlint: Permission denied
husky - commit-msg hook exited with code 126 (error)`

Reply:
probably because the current commit message doesn't fit the conventional commit standards. you can add `--no-verify` to `git commit` or update the commit message

Reply:
solved: `git commit --amend --no-verify`